### DISCUSSION TITLE
集成chatgpt进行翻译

### DISCUSSION THREAD

**Discussion by wangwenxiaos** [YiiGuxing/TranslationPlugin#3466] [Ideas]
希望可以加入让chatgpt进行翻译。我测试chatgpt的翻译准确性还是超级高的。比目前所有的翻译器都高

**Comment by YiiGuxing** [comment]
有在做呢：#3277

**Reply by JoJoWuBeBetter** [reply]
可以在openAI接口里面添加一个自定义地址吗？openAI接口到期后续费很麻烦，使用的都是中转的方式
![image](https://github.com/YiiGuxing/TranslationPlugin/assets/30955722/a00badd0-7117-4df9-adfd-63e383532460)

**Reply by YiiGuxing** [reply]
已经有相关需求请求了呢：#3765 ，想要了解相关动态，可以关注一下
